src/pages/SignUp/SignUp.css

```diff
@@ -0,0 +1,26 @@
+.signup-section{
+    height: 91vh;
+}
+.signup-main-container{
+    padding: 1rem 3rem;
+    box-shadow: 5px 5px 10px rgba(2, 129, 149, 0.3);
```

Comment:
If the value of box-shadow at multiple places is constant, then you can create a variable for this property as well.

Reply:
okk will do that in refactoring.